x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect, useState, useContext, useCallback } from 'react';
+import cytoscape from 'cytoscape';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButtonEmpty,
+  EuiDescriptionList,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiFlyoutBody,
+  EuiPortal,
+  EuiTitle,
+} from '@elastic/eui';
+import { CytoscapeContext } from './cytoscape';
+import { JOB_MAP_NODE_TYPES } from '../../../../../../common/constants/data_frame_analytics';
+// import { DeleteButton } from './delete_button';
+
+interface Props {
+  analyticsId: string;
+  details: any;
+  getNodeData: any;
+}
+
+// TODO move list items to shared place as we use them in the details bit of the wizard
+export interface ListItems {
```

Comment:
I recall this type is available in EUI. Try `EuiDescriptionListProps['listItems']`